shared/src/question-generators/propositional-logic/minimize.ts

```diff
@@ -0,0 +1,196 @@
+import {
+  FreeTextFeedbackFunction,
+  FreeTextFormatFunction,
+  FreeTextQuestion,
+  QuestionGenerator,
+} from "@shared/api/QuestionGenerator"
+import { serializeGeneratorCall } from "@shared/api/QuestionRouter"
+import {
+  getTypingAids,
+  getTypingAidsVars,
+  variableNames,
+} from "@shared/question-generators/propositional-logic/utils"
+import { _ } from "@shared/utils/generics"
+import {
+  compareExpressions,
+  generateRandomExpression,
+  ParserError,
+  PropositionalLogicParser,
+  SyntaxTreeNodeType,
+  tokenToLatex,
+} from "@shared/utils/propositionalLogic"
+import { GetMinimalNormalForm } from "@shared/utils/propositionalLogicMinimize.ts"
+import Random from "@shared/utils/random"
+import { t, tFunctional, Translations } from "@shared/utils/translations"
+
+const translations: Translations = {
+  en: {
+    name: "Minimize boolean expression",
+    description: "Minimize the normal form of a boolean expression",
+    param_size: "The maximal number of variables used",
+    text: "Given the boolean expression \\[\\varPhi={{0}}\\] Compute  the minimized **{{1}}** of $\\varPhi$ labeled as $\\varPhi^*$.",
+    freetext_feedback_parse_error: "Your answer couldn't be parsed.",
```

Comment:
just as a side note: I would shorten the names from `freetext_feedback_*` to `ff_*` just to shorten the lines.